roles/atmosphere/templates/cluster_role_binding.yml

```diff
@@ -9,5 +9,5 @@ subjects:
     namespace: openstack
 roleRef:
   kind: ClusterRole
-  name: atmosphere
+  name: cluster-admin
```

Comment:
what's the reasoning behind this?  if we give atmosphere `cluster-admin` then all the RBAC stuff we do is meaningless

Reply:
@mnaser we install capi in this operator and it means it requires permissions(at least get, patch, create) for resource types  required to install capi. It includes many resource types such as rbac stuff. So i thought it is better to use cluster-admin